INSTALL

```diff
@@ -50,14 +50,19 @@ should install the RPM or other packaging to get all of the stuff you need.
 RPM
 ===
 
-In the 'packaging' dir, there is some crude scripting to help generate RPMs.
-You should be able to just run the following to create an RPM for
-RHEL7/CentOS7:
+In the 'packaging' dir, there is some crude scripting to help generate RPMs
+suitable for RHEL/CentOS 7-8 and Fedora 33-34. Run the following to create such
+an RPM:
 
-  $ ./packaging/rules rpm7
+  $ ./packaging/rules rpm
 
 See './packaging/rules help' for other targets.
 
+On RHEL/CentOS 7-8, our packaging depends on packages in EPEL. On RHEL/CentOS
```

Comment:
I nearly forgot; do we need epel on rhel8 ? your original text mentioned just the powertools repo for rhel8 specifically; I assumed we'd still need also need epel enabled separately, but is that not true?